packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec

```diff
@@ -1,23 +1,16 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
 %global gem_name foreman_scc_manager
 %global plugin_name scc_manager
-%global foreman_min_version 3.4
-%global katello_min_version 3.16.0
```

Comment:
While this was removed here, there is still a `BuildRequires: rubygem(katello) >= 3.16` (and later an automatically generated `Requires: rubygem(katello) >= 3.16`) in the spec coming via https://github.com/ATIX-AG/foreman_scc_manager/blob/a84d26d6aeb11b03b52c290c515b64ef6794066b/foreman_scc_manager.gemspec#L26

(this was "recently" introduced in https://github.com/ATIX-AG/foreman_scc_manager/commit/414226edd690717ef590a166a055bb6e510f1552 and I guess the RPM spec predates that change by lot)